docs/content/querying/segmentmetadataquery.md

```diff
@@ -86,3 +87,21 @@ The grammar is as follows:
 ``` json
 "toInclude": { "type": "list", "columns": [<string list of column names>]}
 ```
+
+### analysisTypes <a name="analysisTypes"></a>
```

Comment:
FYI, markdown creates all lowercase anchors, so you can refer to it using `analysistypes`, similar to http://druid.io/docs/latest/querying/segmentmetadataquery.html#toinclude